src/Widgets/SettingsPopover.vala

```diff
@@ -13,14 +13,15 @@ public sealed class Terminal.SettingsPopover : Gtk.Popover {
 
         set {
             terminal_binding.source = value;
-            if (value != null) {
-                insert_action_group ("term", value.get_action_group ("term"));
-            }
+            //TODO Do we need to do this in Gtk4?
+            // if (value != null) {
+            //     insert_action_group ("term", value.get_action_group ("term"));
+            // }
         }
     }
 
     private const string STYLE_CSS = """
-        .color-button radio {
+        .theme-button radio {
```

Comment:
`color-button` is actually a style class in the system stylesheet, so you might lose some styles from the system by changing this

Reply:
Reverted